ts/src/bitteam.ts

```diff
@@ -0,0 +1,2395 @@
+                feesByNetworkId = withdrawCommissionFixed;
+            }
+            const statuses = this.safeValue (statusesResponse, numericId, {});
+            const deposit = this.safeValue (statuses, 'depositStatus');
+            const withdraw = this.safeValue (statuses, 'withdrawStatus');
+            const networkIds = Object.keys (feesByNetworkId);
+            const networks = {};
+            const networkPrecision = this.safeInteger (currency, 'decimals');
+            for (let j = 0; j < networkIds.length; j++) {
+                const networkId = networkIds[j];
+                const networkCode = this.networkIdToCode (networkId);
+                const networkFee = this.safeNumber (feesByNetworkId, networkId);
+                networks[networkCode] = {
+                    'id': networkId,
+                    'network': networkCode,
+                    'deposit': deposit,
+                    'withdraw': withdraw,
+                    'active': active,
+                    'fee': networkFee,
+                    'precision': networkPrecision,
+                    'limits': {
+                        'amount': {
+                            'min': undefined,
+                            'max': undefined,
+                        },
+                        'withdraw': {
+                            'min': this.parseNumber (minWithdraw),
+                            'max': this.parseNumber (maxWithdraw),
+                        },
+                        'deposit': {
+                            'min': this.parseNumber (minDeposit),
+                            'max': undefined,
+                        },
+                    },
+                    'info': currency,
+                };
+            }
+            result[code] = {
+                'id': id,
+                'numericId': numericId,
+                'code': code,
+                'name': code,
+                'info': currency,
+                'active': active,
+                'deposit': deposit,
+                'withdraw': withdraw,
+                'fee': fee,
+                'precision': precision,
+                'limits': {
+                    'amount': {
+                        'min': undefined,
+                        'max': undefined,
+                    },
+                    'withdraw': {
+                        'min': this.parseNumber (minWithdraw),
+                        'max': this.parseNumber (maxWithdraw),
+                    },
+                    'deposit': {
+                        'min': this.parseNumber (minDeposit),
+                        'max': undefined,
+                    },
+                },
+                'networks': networks,
+            };
+        }
+        return result;
+    }
+
+    async fetchOHLCV (symbol: string, timeframe = '1m', since: Int = undefined, limit: Int = undefined, params = {}): Promise<OHLCV[]> {
+        /**
+         * @method
+         * @name bitteam#fetchOHLCV
```

Comment:
can we add the link to the docs here?

Reply:
The exchange's members told about this endpoint, there's no description of it in the exchange's docks.